src/components/backend-ai-storage-list.ts

```diff
@@ -2781,10 +2805,77 @@ export default class BackendAiStorageList extends BackendAIPage {
       this.notification.text = PainKiller.relieve(err.title);
       this.notification.detail = err.message;
       this.notification.show(true, err);
-      indicator.end(1000);
+      indicator.end(100);
     });
   }
 
+  _executeSSHProxyAgent() {
+    if (this.volumeInfo[this.vhost]?.sftp_scaling_groups?.includes('ssh')) {
+      if (this.systemRoleSupportedImages.length > 0) {
+        this._launchSystemRoleSSHSession();
+        this._toggleSSHSessionButton();
+      } else {
+        this.notification.text = _text('data.explorer.NoImagesSupportingSystemSession');
+        this.notification.show();
+      }
+    } else {
+      this.notification.text = _text('data.explorer.SftpScalingGroupsDoNotExist');
+      this.notification.show();
+    }
+  }
+
+  /**
+   * Launch system role filebrowser image and open the dialog that includes the ssh link.
+   */
+  async _launchSystemRoleSSHSession() {
+    let sessionId;
+    const imageResource: Record<string, unknown> = {};
+    const configSshImage = globalThis.backendaiclient._config.systemSSHImage;
+    const images = this.systemRoleSupportedImages.filter((image: any) => (image['name'].toLowerCase().includes('filebrowser') && image['installed']));
+    // TODO: use lablup/openssh-server image
+    // select one image to launch system role supported session
+    const preferredImage = configSshImage.length > 0 ? configSshImage : images[0];
+    const environment = preferredImage['registry'] + '/' + preferredImage['name'] + ':' + preferredImage['tag'];
```

Comment:
since configSshImage is a simple string, but It seems like it's been treated as an object.

Reply:
oh, it's my mistake. bull's eye!